### PR TITLE
Fixed formatting in README

### LINKED ISSUE #20: Cleanup README
The README looks very weird and is badly formatted.

### PR DESCRIPTION
**Pull request by VoyTechnology**
Resolves #20 

This change includes no updates to the documentation itself, just updates the formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,168 +27,221 @@ Then open [localhost:8000/rrs.cgi](http://localhost:8000/rrs.cgi)
...
 ### Renew User
 
-1. Queries RB LDAP using user nickname.
-2. If Yearspaid <1, set yearsPaid=1
-3.  *problem* if the user's shell is /usr/local/shell/expired and is renewed, shell isn't reset to /usr/local/shell/zsh
-4.  *problem* if renewing && usertype is associat/committe it will hint at restoring usertype to member
-	- For committe this wouldn't be as bad, but for associat this poses an issue frequently.
+1. [x] Queries RB LDAP using user nickname.
+2. [x] Set yearsPaid=1 Set yeats paid to 1 if less than 1
+3. [ ] Reset user shell from `/usr/local/shell/expired` back to previous user's shell
```
Comment (butlerx) [inline README.md]: this is done

File: README.md [hunk 2]
```diff
@@ -22,168 +27,221 @@ Then open [localhost:8000/rrs.cgi](http://localhost:8000/rrs.cgi)
...
+PostgresSQL version 7.2 or higher must be installed. Details on doing this vary
+depending on the operating system and is outside the scope of this document
+however, full instructions can be found on the PostgresSQL website.
 
-   PostgresSQL version 7.2 or higher must be installed. Details on doing this vary depending on the operating system and is outside the scope of this document however, full
-   instructions can be found on the PostgresSQL website.
+##### Python
 
-    Python
+Python version 2.2 or higher must be installed. Details on doing this vary
```
Comment (butlerx) [inline README.md]: this is python3 now

File: README.md (hunk 1 above)
Comment (VoyTechnology) [reply README.md]: I am aware, like I said, just formatting. Can update this in a separate PR.

File: README.md (hunk 2 above)
Comment (VoyTechnology) [reply README.md]: I am aware, like I said, just formatting. Can update this in a separate PR

**Comment by butlerx** [comment]
If there was lots in need of updateing id agree but since its 3 character i think its just as good to add it to this pr

**Comment by VoyTechnology** [comment]
@butlerx PTAL

**Review by butlerx — APPROVED** [review]

**Pull request merged**